src/pages/[platform]/deploy-and-host/fullstack-branching/branch-deployments/index.mdx

```diff
@@ -47,26 +47,69 @@ You can also define a pattern to connect only certain branches. For example, set
   <source src="/images/gen2/fullstack-branching/autobranch.mp4" />
 </video>
 
-3. Push a commit to your `featureA` and `staging` branches that match the pattern. You should start seeing deployments on the console page. You will now have three fullstack branches deployed.
+3. Push a commit to your `feature/A` and `staging` branches that match the pattern. You should start seeing deployments on the console page. You will now have three fullstack branches deployed.
 
-![Production, featureA, and staging branches listed on app overview page in Amplify console.](/images/gen2/fullstack-branching/branches.png)
+![Production, feature/A, and staging branches listed on app overview page in Amplify console.](/images/gen2/fullstack-branching/branches.png)
 
 ## Promote changes to production
 
 In Gen 2, promoting changes to production follows the normal Git-based workflow.
 
-![Workflow for merging changes from featureA branch to main, or production, branch.](/images/gen2/fullstack-branching/gitflow.png)
+![Workflow for merging changes from feature/A branch to main, or production, branch.](/images/gen2/fullstack-branching/gitflow.png)
 
-1. Make a change in your `featureA` branch.
+1. Make a change in your `feature/A` branch.
 
-```terminal
-> git checkout featureA
+```sh showLineNumbers={false}
+git checkout -b feature/A
 
 ## make some edits to your code
 
-> git commit --am "New data model to track comments for todos added"
+git commit --am "New data model to track comments for todos added"
 
-> git push origin featureA
+git push origin feature/A
 ```
 
 2. Submit a pull request to your `main` branch. Once your team has validated the changes, merge the pull request to `main`. This will initiate a build on your `main` branch and update any frontend or backend resources that you changed.
+
+
+### Generate client config
+
+You can generate the config for a branch environment by running:
+
+```bash showLineNumbers={false}
+npx amplify generate config --app-id <AMPLIFY_APP_ID> --branch main --format=["mjs", "json", "json-mobile", "ts", "dart"] --out-dir=path/to/config
```

Comment:
```suggestion
npx amplify generate config --app-id <your-amplify-app-id> --branch main --out-dir path/to/config
```

nit for placeholder text https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md#16-bracket-replacement-text-in-codeblocks

also format will soon be just `json` and `dart`, and the equal sign is optional, but for consistency we can remove them